grafana_client/elements/team.py

```diff
@@ -152,3 +152,38 @@ def update_team_preferences(self, team_id, preferences):
         update_team_preferences_path = "/teams/%s/preferences" % team_id
         r = self.client.PUT(update_team_preferences_path, json=preferences)
         return r
+
+    def get_team_external_group(self, team_id):
+        """
+        The Data Source Permissions is only available in Grafana Enterprise.
```

Comment:
I think a copy/paste error slipped in here. Can you adjust the docstring comments correspondingly?